docs/metrics-analysis-prometheus/prometheus.md

```diff
@@ -0,0 +1,55 @@
+# Prometheus
+
+## Introduction
+
+Prometheus is integrated with our `gh_actions_exporter` repository,
```

Comment:
Prometheus is not integrated. The user is expected to have an existing prometheus installation and configure it to pull metrics from the available endpoint. 

Specify where the metrics are exposed

Reply:
When configuring Prometheus to pull metrics from an exporter, you need to specify the url. The `url` itself will depend on the user's install but the `/metrics` path will always be the same, so add this information to help users.

Reply:
Done!